.github/autobump.txt

```diff
@@ -2187,6 +2187,7 @@ zellij
 zenith
 zeromq
 zf
+zig
```

Comment:
Zig isn't really a good candidate for autobump, because it usually requires multiple changes during version bumps

Reply:
i think we can just get the bump PR (which is automated), and put changes in the PR.

Reply:
That means contributors can't use `brew bump[-formula-pr]` to work on an update themselves though